### DISCUSSION TITLE
Daly BMS not shown under load

### DISCUSSION THREAD

**Discussion by jannikweichert** [Louisvdw/dbus-serialbattery#853] [Q&A]
I have a Daly BMS connected via RS485 -> USB & UART -> USB to my Cerbo GX. Both connections are working fine in idle and with lower load. But above ~ 500W the battery suddenly disappears from the Dashboard Page & Device List. As soon as I am under the threshold, battery immediately reappears. 

Any idea what I can check?

**Comment by mr-manuel** [comment]
1. How long are the cables?
2. Is the data cable near the power cables?
3. Is the data cable shielded?

**Reply by jannikweichert** [reply]
1. Cable is ~1 meter
2. Yes, indeed
3. No, not shielded (or are normal usb extension cables shielded?)

Sounds stupid now in summary 🙈

Is there a shielding option that allows the data cable to be right next to the power cable? Would be hard to increase the distance as I am using the same whole in my vehicle floor. 

Thank you!

**Reply by mr-manuel** [reply]
First you could make a test and attach a USB cable that is not near the power cables, so you see if that is the problem.
